bundles/org.openhab.binding.panamaxfurman/src/main/resources/OH-INF/i18n/panamaxfurman.properties

```diff
@@ -0,0 +1,24 @@
+binding.panamaxfurman.name = PanamaxFurman binding
+binding.panamaxfurman.description = Binding for Panamax/Furman power conditioners which have the BlueBOLT-CV1 or BlueBOLT-CV2 interface card installed.
```

Comment:
Please use the mvn command to add default translations.
some  of this file are filled, some not.